Makefile

```diff
@@ -174,6 +190,12 @@ else
 	$(CXX) $(CPPFLAGS) $(CXXFLAGS) $(DEFS) $(INCLUDES) -fPIC -shared -o $@ $(SOURCES) $(LIBS)
 endif
 
+ifeq ($(OS_TAG),linux)
+	$(STRIP) --only-keep-debug $@ -o build/sym/$(@F).debug
+	$(STRIP) -g $@
+	$(OBJCOPY) --add-gnu-debuglink=build/sym/$(@F).debug $@
+endif
```

Comment:
Is it the right place to extract debuginfo? I think, for development purposes, one binary with debug symbols is more convenient; we just do redundant steps here. Should we strip binaries at the packaging step instead?